Makefile

```diff
@@ -532,6 +532,9 @@ docker-compose-sources: .git/hooks/pre-commit
             $(EXTRA_SOURCES_ANSIBLE_OPTS)
 
 docker-compose: awx/projects docker-compose-sources
+	ansible-galaxy install -r tools/docker-compose/ansible/requirements.yml;
+	ansible-playbook -i tools/docker-compose/inventory tools/docker-compose/ansible/initialize_containers.yml \
+	  -e enable_vault=$(VAULT);
```

Comment:
What if we had a new target, like `make docker-compose-plumb` which runs plumbing playbooks for all of the env vars which are set up?

Reply:
Yea, I've been trying to think about how to do the plumbing. A make target wouldn't be a bad idea.

Reply:
If ya'll go this direction, please put it in a script if it's more than one line. You can export all the 'make' vars to the script if that's why you keep using make as a harness. Basically, extract any complex command blocks to a specialized script that can be run outside of make, too.